UI/SidePanel.cpp

```diff
@@ -487,6 +488,10 @@ class SidePanel::PlanetPanel : public GG::Control {
       * returns the id of the clicked planet */
     mutable boost::signals2::signal<void (int)> LeftClickedSignal;
 
+    /** emitted when the planet is left double clicked by the user.
```

Comment:
What's the difference between LeftDoubleClickedSignal and PlanetLeftDoubleClickedSignal?